administration/configuring-fluent-bit/classic-mode/configuration-file.md

```diff
@@ -2,6 +2,8 @@
 description: This page describes the main configuration file used by Fluent Bit
 ---
 
+<img referrerpolicy="no-referrer-when-downgrade" src="https://static.scarf.sh/a.png?x-pxid=5e67142e-3887-4b56-b940-18494bcc23a7" />
```

Comment:
We should be consistent in where we add the image. Since it follows the h1 in most of the modified files, it should continue to be after the h1 in all of the files.